packages/babel-plugin-transform-regenerator/package.json

```
@@ -7,7 +7,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-plugin-transform-regenerator",
   "main": "lib/index.js",
   "dependencies": {
-    "regenerator-transform": "^0.14.0"
+    "regenerator-transform": "^0.14.2"
```

Seems like https://github.com/facebook/regenerator/pull/377 is available in `regenerator-transform:0.14.1` shouldn't this be 0.14.1?

```suggestion
    "regenerator-transform": "^0.14.1"
```

---

It isn't: https://github.com/facebook/regenerator/commits/master
- facebook/regenerator@ffc74ccb098e30c28c374fd114ce9ea9cb5614fe
- facebook/regenerator@d4ccc3ba278753c7c37e2db2147834007b9075b1

---

You're right. Sorry. My bad.